### PR TITLE
Catalyst set up middleware article

### PR DESCRIPTION
**Pull request by oliverheywood451**

### CODE REVIEW

File: content/documents/api-from-scratch.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+type:           tutorial
+title:          Start .NET middleware from Scratch
```
Comment (robertsoniv) [inline content/documents/api-from-scratch.mdx:3]: Rename your file to match the title as much as possible... something like:
`start-dotnet-middleware-from-scratch.mdx`

File: content/documents/api-from-scratch.mdx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+
+### Publish API to Azure App Service 
+
+You will need to create a new App Service resource in azure. Then add an app setting for `APP_CONFIG_CONNECTION` in the Configuration tab the same way you did locally in Visual Studio. Once that is done, you're ready to deploy code. Deploying is its own devops discipline. For the quick and dirty purpose of getting started you can [deploy directly from Visual Studio](https://docs.microsoft.com/en-us/aspnet/core/tutorials/publish-to-azure-webapp-using-vs?view=aspnetcore-5.0). However, a CI/CD deployment tool like [Azure Devops](https://azure.microsoft.com/en-us/services/devops) can greatly improve your deploy processes. 
+
+Azure Devops is our recommended approach. This project includes a deploy.yaml (link to come) file that specifies an Azure Devops build and release pipeline. Modify that file with your Azure App Service name and push those changes to the banch you want to deploy off. Then, in Azure Devops create new build pipeline. Specify your git repository and Azure Devops should find deploy.yaml and handle the pipeline set up.
+
+![Alt text](./where_is_your_code.png "New Azure devops Pipeline")
```
Comment (robertsoniv) [inline content/documents/api-from-scratch.mdx:60]: Again, do not store image files in `/content/documents`

**Review by robertsoniv — CHANGES_REQUESTED** [review]
See review comments for requested changes

File: content/documents/api-from-scratch.mdx [hunk 3]
```diff
@@ -0,0 +1,62 @@
+---
+type:           tutorial
+title:          Start .NET middleware from Scratch
+description:    Step by step instructions to stand up a hosted middleware using .NET Catalyst
+author:         oheywood
+publishDate:    2021-03-10
+updatedDate:    
+tags:           ["Middleware", "Catalyst", ".NET"]
```
Comment (robertsoniv) [inline content/documents/api-from-scratch.mdx:8]: Consider adding the tag "Integrations"

**Comment by oliverheywood451** [comment]
Updated PR based on comments

**Review by robertsoniv — APPROVED** [review]
Awesome! Thanks Oliver

**Pull request merged**